mycroft/client/speech/silence.py

```diff
@@ -0,0 +1,402 @@
+"""Silence detection using Silero VAD."""
```

Comment:
this file was copy pasted from mycroft, it should have a regular mycroft header i suppose?

the comment itself is no longer valid since we use OPM not hardcoded silero

Reply:
Ah, I assumed they had that in some automation, so I thought you extracted this from something else.. idk then, maybe it's better to leave it like this to avoid a merge conflict if we pull their changes later?